src/Encoding.js

```diff
@@ -194,14 +194,6 @@ var Encoding = module.exports = (function() {
     return sort;
   };
 
-  proto.any = function(f) {
```

Comment:
Have you check that these `Encoding.all/any` aren't being used in any dependent projects?